gemfiles/rails_head.gemfile

```diff
@@ -12,6 +12,7 @@ group :development, :test do
   gem "sqlite3", platform: [:ruby, :mswin, :mingw]
   gem "activerecord-jdbcsqlite3-adapter", "~> 1.3.13", platform: :jruby
   gem "test-unit", "~> 3.0"
+  gem "rackup"
```

Comment:
rack 3.x no longer includes `Rack::Server`, see https://github.com/rack/rack/blob/main/UPGRADE-GUIDE.md#binrackup-rackserver-rackhandlerand--racklobster-were-moved-to-a-separate-gem